tests/config-test.js

```diff
@@ -204,10 +229,49 @@ describe('src/config', function () {
           creditLimit: 5,
           resetSeconds: 60,
           actorField: 'ip',
+          matchPolicy: 'stop',
           label: 'post-by-ip',
           comment: '5 rpm for any POST, by ip',
         }, rule);
       });
+
+      it('with a canary rule', function () {
+        const rules = config.findRules({
+          method: 'GET',
+          path: '/ping',
+          local: 'true',
+          ip: '1.2.3.4',
+        });
+        assert.equal(2, rules.length);
+        assert.deepEqual([
```

Comment:
Recommend `assert.deepStrictEqual`: `assert.deepEqual` [has been deprecated](https://nodejs.org/api/assert.html#assert_assert_deepequal_actual_expected_message)... see old versions of node:

```node
> assert.deepEqual([], {})
undefined
> assert.deepEqual([], {wat: true})
AssertionError: [] deepEqual { wat: true }
    at repl:1:8
    at REPLServer.defaultEval (repl.js:248:27)
    at bound (domain.js:280:14)
    at REPLServer.runBound [as eval] (domain.js:293:12)
    at REPLServer.<anonymous> (repl.js:412:12)
    at emitOne (events.js:82:20)
    at REPLServer.emit (events.js:169:7)
    at REPLServer.Interface._onLine (readline.js:210:10)
    at REPLServer.Interface._line (readline.js:549:8)
    at REPLServer.Interface._ttyWrite (readline.js:826:14)
```

Reply:
Also legit to punt that to a future refactor PR, if you feel inclined.